docs/develop/sso-in-office-add-ins.md

```diff
@@ -91,7 +91,7 @@ The following is an example of the markup:
 
 Add JavaScript to the add-in to:
 
-* Call [getAccessToken](/javascript/api/office/officeruntime.auth#getAccessToken-options--callback-).
+* Call [getAccessToken](/javascript/api/office-runtime/officeruntime.auth#getAccessToken-options-).
```

Comment:
bookmark portion of the link target needs to be all lowercase -> `#getaccesstoken-options-`